src/Utilities/cachingoptimizer.jl

```diff
@@ -397,6 +401,20 @@ function MOI.set!(m::CachingOptimizer, attr::Union{MOI.AbstractVariableAttribute
     MOI.set!(m.model_cache, attr, index, value)
 end
 
+# Names are not copied, i.e. we use the option `copynames=false` in
+# `attachoptimizer`, so the optimizer do not need to support names.
```

Comment:
clarify: so the caching optimizer can support names even if the optimizer doesn't.